src/components/AvatarGroup/AvatarGroup.tsx

```diff
@@ -50,23 +54,13 @@ const AvatarGroup: React.FC<AvatarGroupProps> = ({
   max = 5,
   counterProps,
   counterTooltipCustomProps,
-  counterTooltipIsVirtualizedList = false
+  counterTooltipIsVirtualizedList = false,
+  padding = true
 }) => {
   const { displayAvatars, counterTooltipAvatars } = useMemo(() => {
     const childrenArray = Array.isArray(children) ? children : [children];
```

Comment:
Nice, let's just add null check here
```suggestion
    if (!children) {
      return {};
    }
    const childrenArray = Array.isArray(children) ? children : [children];
```